crates/ruff_python_formatter/src/expression/expr_list.rs

```diff
@@ -18,6 +18,39 @@ impl FormatNodeRule<ExprList> for FormatExprList {
             ctx: _,
         } = item;
 
+        let comments = f.context().comments().clone();
+        let dangling = comments.dangling_comments(item.into());
+
+        // The empty list is special because there can be dangling comments, and they can be in two
+        // positions:
+        // ```
```

Comment:
Same here, can we try to add `python` for consistency so as to disambiguate from actual Rust in the future?

Reply:
checked all the places with triple backticks